### DISCUSSION TITLE
V3+ styles are missing unless resave tailwind.config.js

### DISCUSSION THREAD

**Discussion by weilinzung** [tailwindlabs/tailwindcss#6963] [Help]
**What version of Tailwind CSS are you using?**

3.0.2

**What build tool (or framework if it abstracts the build tool) are you using?**

Webpack 5.26.3

**What version of Node.js are you using?**
14.17.3

**What browser are you using?**

 Chrome

**What operating system are you using?**

macOS

**Describe your issue**

When working on a template that adds some new tailwind classes which have never been used, we would have to resave `tailwind.config.js` to in-order get the new class styles showing. Is this related to cache that with tailwind & WebpacK? In v2, never seems to have that issue.

**Comment by reinink** [comment]
Hey! So without a minimal reproduction (either via Tailwind Play or a Git repo), it's very hard for us to help in these situations.

I suspect this might have something to do with your `content` config in your `tailwind.config.js` file, but without seeing it I can't say. I'd also recommend getting on the latest version of Tailwind CSS (v3.0.12) to see if that helps 👍 

If you're able to provide a minimal reproduction, I'd be happy to have another look!

**Reply by arthurdenner** [reply]
Hi @reinink! 👋🏽 

I don't have an example with bare Webpack but I had the same issue in Remotion - that uses Webpack under the hood.
I have a reproducible example at https://github.com/arthurdenner/remotion-tailwind-v3-test
The steps on the README should result in the same issue. Tailwind v2 works fine though.

Here is a comment with a video showing the issue: https://github.com/remotion-dev/remotion/issues/737#issuecomment-1007553694.

Looking at the discussions, this is probably related to https://github.com/tailwindlabs/tailwindcss/discussions/6570, since Storybook also uses Webpack under the hood.

Let me know if you prefer a bare example, I can try to create one.

**Reply by weilinzung** [reply]
Hi @reinink

Have you gotten a chance to check @arthurdenner 's example?

Thanks!

**Comment by weilinzung** [comment]
Hi @reinink.

We just have a very simple config. I will create a simple repo for testing.

```
module.exports = {
  content: ['./src/**/*.{scss,js,html,md,yml}'],
  plugins: [
    require('@tailwindcss/typography'),
    require('@tailwindcss/aspect-ratio'),
    require('@tailwindcss/forms')({
      strategy: 'class',
    }),
  ],
};
```
